docs/source/models/gbd2017/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -39,10 +39,10 @@ on the ages and sexes to which the cause applies.
      -
    * - YLL only
      - False
-     -
+     - Severe CKD only for YLL
    * - YLD only
      - False
-     -
+     - Moderate CKD, Severe CKD include YLD
    * - YLL age group start
```

Comment:
These notes are a little unclear to me - I think more detail might be helpful here!

Reply:
Thanks! I'll add.

Reply:
@alibow I ended up removing this information from the Restrictions table and added more explanation to the 'Vivarium Modeling Strategy' section instead. Please let me know if it's a clearer way to frame and has sufficient detail! Thank you.

Reply:
@alibow I'm going to go ahead and merge my branch to master but can create a new PR if we find my update isn't descriptive enough. I'll reach out to you in coworking today!